packages/admin-panel/src/VizBuilderApp/components/MapOverlay/MapOverlayMetadataForm.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import { Autocomplete, TextField } from '@tupaia/ui-components';
+import { useCountries, usePermissionGroups, useProjects } from '../../api/queries';
+import { useVizConfig } from '../../context';
+import Chip from '@material-ui/core/Chip';
+
+export const MapOverlayMetadataForm = ({ Header, Body, Footer, onSubmit }) => {
+  const { handleSubmit, register, errors } = useForm();
+  const [{ visualisation }, { setVisualisationValue }] = useVizConfig();
+  const {
+    data: permissionGroups = [],
+    isLoading: isLoadingPermissionGroups,
+  } = usePermissionGroups();
+  const { data: allProjects = [], isLoading: isLoadingAllProjects } = useProjects();
+  const { data: allCountries = [], isLoading: isLoadingAllCountries } = useCountries();
+
+  // Save the default values here so that they are frozen from the store when the component first mounts
+  const [defaults] = useState(visualisation);
+  const {
+    name,
+    code,
+    mapOverlayPermissionGroup,
```

Comment:
Is there a reason we call this `mapOverlayPermissionGroup` instead of just `permissionGroup`? There's an existing `permissionGroup` field that's expected for vizes as the `report` requires one. The `permissionGroup` for the map overlay should be consistent with this one, so we could just have a single `permissionGroup` field that gets saved in both objects?

Regardless, it looks like there's minor issue with calling this `mapOverlayPermissionGroup` here that the UI doesn't show the permission group correctly for map overlays.
![image](https://user-images.githubusercontent.com/59544282/142957985-03604e1e-44a5-4f84-a138-64135fdeb4ff.png)

Reply:
Discussed. We want to leave is as two separate fields for now. In future cleanup ticket probably make consistent with API in having a viz object like { report: ..., mapOverlay: ... }. 
But, for now fix:
- Make sure export spits out viz.reportPermissionGroup (doesn't look like it is)
- Fix UI
- Make sure create flow populates both when you select it in the edit form modal

Reply:
Fixed 2 and 3.

1. was happening because it was a legacy report. I have checked and it works correctly for non-legacy map overlay vizes.